model/openapi.yml

```diff
@@ -808,7 +814,8 @@ paths:
     get:
       operationId: status
       parameters:
-        - $ref: '#/components/parameters/requestId'
+        - $ref: "#/components/parameters/requestId"
+        - $ref: "#/components/parameters/apiVersion"
```

Comment:
As we're adding the api version middleware, all endpoint including status can now return a 400, we should add that to the spec